kafka/source_config.go

```diff
@@ -118,5 +129,26 @@ func NewSourceConfig(hosts []string, topic string, consumerGroupId string) Sourc
 	out.ReadBackoffMinMs = 100
 	out.ReadBackoffMaxMs = 1000
 	out.MaxAttempts = 5
+	out.ConfiguredOutputEntryFunc = ValueOutputEntryFunc
```

Comment:
Please give it a more standard (base on the lib grammar) name such as ValueExtractor

Reply:
ok